app/controllers/comment_controller.php

```diff
@@ -0,0 +1,68 @@
+<?php
+
+class CommentController extends AppController 
+{
+    public function edit()
+    {
+        $user = new User($_SESSION['username']);
+        $check = Param::get('check', false);
+        $title = " | Edit comment";
+        $params = array(
+            'comment_id' => Param::get('id', ''),
+            'thread_id'  => Param::get('thread_id', ''),
+            'user_id'    => $user->user_id,
+        );
+        $comment = new Comment($params);
+
+        $comment_content = Comment::getCommentContent(Param::get('id', 0));
+        $comment->error = isset($comment_content->error) ? $comment_content->error : null;
+        if (isset($comment->error)) {
```

Comment:
Sir as you can see I have a default value of null at the end of the ternary operator, and having a value of null in variable will return false if checked in isset() function. Thanks sir! :bow: